upa/contracts/UpaProofReceiver.sol

```diff
@@ -41,6 +41,7 @@ error TooManyPublicInputs();
 error MaxNumPublicInputsTooLow();
 error TooManyCommitmentPoints();
 error InconsistentPedersenVK();
+error DummyProofInSubmission();
```

Comment:
This is for 3.3, isn't it? I think 3.3 is a non-issue, because it's the worker's job to skip any submissions with the dummy proofId in them (and challenges with the dummy proofId are forbidden). 

I removed everything related to this check in https://github.com/NebraZKP/upa/pull/30

Let me know if you disagree.

Reply:
I'm in two minds.  I can see that we could get away without it and I think it's fair that the aggregator receives fees in this case, but part of me is much more comfortable with just simple, well-defined behavior w.r.t. this, rather than letting it fall-through.

Let's discuss offline.